Header header          #standard header information

ndt_map/NDTVectorMapMsg ndt_maps
grid_map_msgs/GridMap prior
